sources/packages/web/src/components/layouts/FullPageContainer.vue

```diff
@@ -11,7 +11,7 @@
     <v-container :fluid="fullWidth">
       <template v-if="layoutTemplate === LayoutTemplates.CenteredCard">
         <v-row justify="center">
-          <v-card class="mt-4 p-4 w-100" :class="widthClass">
+          <v-card class="mt-4 p-4 w-100 overflow-visible" :class="widthClass">
```

Comment:
Do we need to add this overflow property at container level? because we set the overflow property in different places depending upon the need. I know the default property of overflow is visible but if that is not happening do we need to force it at container level?

![image](https://github.com/bcgov/SIMS/assets/54600590/7c3553ed-de78-45a7-9522-3a21cb18b47f)

Reply:
Yes, I changed that as we talked about it. Thanks for the help checking that.